src/base/Tabs/Tabs.tsx

```diff
@@ -1,3 +1,37 @@
+// import { Tabs as MuiTabs, type TabsProps as MuiTabsProps } from '@mui/material';
+
+// export function Tabs(props: MuiTabsProps): JSX.Element {
+//   return (
+//     <MuiTabs
```

Comment:
why its being commented and not removed?

Reply:
My apologies, I've removed the same.